h/static/styles/partials-v2/_group-invite.scss

```diff
@@ -9,22 +9,16 @@
 }
 
 .group-invite__input {
-  padding: 5px;
-  padding-top: 0;
   padding-bottom: 0;
+  padding-left: 5px;
+  padding-right: 22px;  /* Make space for the clipboard icon. */
+  padding-top: 0;
   margin-top: 12px;
   margin-bottom: 8px;
   border: 1px solid $grey-3;
   border-radius: 2px;
   width: 100%;
   height: 31px;
-
-  .env-js-capable & {
```

Comment:
I removed these to reduce the number of states to reason about to just two - mobile and non-mobile. Otherwise we'd have 6 to deal with and test (mobile|non-mobile * {JS,no-JS,JS timeout}. In general for this reason, I would suggest that we _only_ add non-JS/timeout selectors if required to make the component usable, not for purely aesthetic reasons.

Otherwise we introduce additional complexity, which increases the likelihood of bugs, which means a worse experience for all our users.